packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -629,6 +629,15 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         this.channel.reSubmit(innerContents.type, innerContents.content, localOpMetadata);
     }
 
+    public async rebaseOp(contents: any, localOpMetadata: unknown) {
+        if (!this.channel) {
+            await this.channelDeferred?.promise;
```

Comment:
Do we always have this.channelDeferred when we get here? It's created only in realize() and I'm not sure it's guaranteed it's called before we get here. So await of this.realize() is much better - this.channelDeferred (I think) should be internal implementation of this.realize method